.evergreen.yml

```diff
@@ -1666,7 +1666,7 @@ buildvariants:
     tags: [ "pr_patch", "staging", "e2e_mco_test_suite" ]
     <<: *community_dependency
     run_on:
-      - ubuntu2004-large
+      - ubuntu2404-large
```

Comment:
`--retry-all-errors` was introduced with [curl 7.71.0](https://curl.se/ch/7.71.0.html)

On 2004, the flag was not found and mco tests failed:
https://parsley-beta.corp.mongodb.com/evergreen/mongodb_kubernetes_e2e_mco_tests_replica_set_arbiter_patch_fd4450bdd5aab857c0b9570cfd1cb4f3b8bb42f8_68f787dc22f3b60007048935_25_10_21_13_17_20/1/task?bookmarks=0,1224

`curl: option --retry-all-errors: is unknown`